### DISCUSSION TITLE
Order of tables in BB Builder

### DISCUSSION THREAD

**Discussion by FloMiLe** [Budibase/budibase#1146] [Ideas]
Just a minor suggestion: I noticed that the order of tables in BB Builder's Table section depends on where a table was inserted upon creation.

I think it would be helpful to either sort the tables alphabetically, or (preferably) to be able to re-arrange the order.

**Comment by joebudi** [comment]
Thanks for the suggestion. I will alert the team regarding this. It shouldn't be that difficult.

**Comment by crudyrudy** [comment]
In fact, the order depends not only on the creation order but also on the last edit. When the name of a column is adjusted, the column is moved to the far right. Inelegant.

**Reply by fire** [reply]
I have also encountered this.
